src/Nncase.Evaluator/Math/Quantize.cs

```diff
@@ -20,6 +21,16 @@ public IValue Visit(IEvaluateContext context, Quantize target)
         var input = context.GetOrtArgumentValue(target, Quantize.Input);
         var quantParam = context.GetArgumentValueAsScalar<QuantParam>(target, Quantize.QuantParam);
         var zeroPoint = Tensor.FromScalar(quantParam.ZeroPoint).CastTo(target.TargetType);
+
+        // only support qint8 in onnx
+        if (input.DataType == OrtDataType.Float && target.TargetType == DataTypes.Int16)
+        {
+            var result = input.ToArray<float>().Select(x => (short)((x / quantParam.Scale) + quantParam.ZeroPoint)).ToArray();
```

Comment:
It seems this line has very low performance as two `ToArray` means memcpy and scalar `Select` cannot be optimized by JIT. Perhaps it's better to use OrtKI to run div + plus and cast to target type.